// This file was automatically generated from instructions.soy.
// Please don't edit this file by hand.

if (typeof scb_assignment_specific_mit706s14ps2 == 'undefined') { var scb_assignment_specific_mit706s14ps2 = {}; }


scb_assignment_specific_mit706s14ps2.page1 = function(opt_data, opt_sb) {
  var output = opt_sb || new soy.StringBuilder();
  output.append('<div class=\'scb_s_mit706s14ps2_assignment_overview\' xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"><h2>Goal</h2>The goal of this assignment is to increase understanding of cell biology techniques and their application by providing you with the opportunity to conduct and analyze data from your own cell biology experiments.<h2>StarCellBio</h2><p>To conduct your cell biology experiments, you will use StarCellBio, a cell and molecular experiment simulator, launched from the MITx 7.06r course website.</p><h2>Due Date & Points</h2>Your answers to the questions on the MITx 7.06r website will be due on <b>Friday, May 2nd by 11 AM</b>.<p/>This assignment contains five questions. To obtain full credit, you will need to select and submit answers to <strong>three out of the five</strong> questions for a total of <strong>30 points</strong>.  <p/>You have the opportunity to submit the two remaining questions for extra credit. Each remaining extra credit question is worth a total of <strong>5 points</strong>. For a question to be considered for extra credit, it must be completed in its entirety. Remaining questions that are partially answered will not be graded for extra credit. <p/>At the top of each question, you will be asked to indicate whether you would like your answers to that question to be considered for full credit, extra credit or neither.</div>');
  return opt_sb ? '' : output.toString();
};


scb_assignment_specific_mit706s14ps2.page2 = function(opt_data, opt_sb) {
  var output = opt_sb || new soy.StringBuilder();
  output.append('<div class=\'scb_s_mit706s14ps2_assignment_overview\' xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"><h2>Background Information</h2><p>Your graduate thesis research is focused on the identification and characterization of novel regulators of the mitotic cell cycle. To identify such novel cycle cell regulators, you initiate a collaboration with the Broad Institute, which will provide access to their extensive RNAi library for an RNAi-based screen.</p><p>Before you screen all of the library’s siRNA constructs, you decide to perform a preliminary screen by treating Drosophila Schneider 2 (S2) cells, a cell line originally derived from Drosophila melanogaster embryos, with a subset of the RNAi library. You identify 5 different siRNA constructs (<b>siRNA #1-5</b>) that result in a cell cycle phenotype and that you would like to further characterize their effect on cell cycle progression.</p><p>For your experimental analyses, you have the following cell cycle specific drugs that provide known, expected results under appropriate conditions to which you may wish to compare your experimental samples involving the RNAi treatments.<ul><li><i>Serum-free media</i> = S2 cells grown in media that lacks serum, and therefore lacks all growth factors, which results in a cell cycle arrest in G1.</li><li><i>Hydroxyurea</i> = a drug that depletes the cell of deoxyribonucleotides (dNTPs), which results in a cell cycle arrest in S-phase.</li><li><i>Colchicine</i> = a drug that inhibits polymerization of microtubules, which results in a cell cycle arrest in mitosis.</li></ul></p><h2>Experimental Analyses</h2><p>To characterize the effects of siRNA #1-5 on cell cycle progression, you perform the following analyses after each RNAi treatment:</p><p>I. You first analyze DNA content by flow cytometry using propidium iodide (PI), a fluorescent dye that binds, or intercalates, DNA.</p><p>II. Next, you analyze several key cell cycle proteins by western blot. For the western blotting analysis, you have the following antibodies available to you, including an antibody against Pgk1, a protein whose levels do not change throughout the cell cycle.<table class="scb_s_objective_table"><thead><td><b>Antibody</b></td><td><b>Protein Detected</b></td><td><b>Expected Molecular Weight (kDa)</b></td></thead><tbody><tr><td>anti-Chk1</td><td>Chk1</td><td>54</td></tr><tr><td>anti-Cyclin B</td><td>Cyclin B</td><td>58</td></tr><tr><td>anti-Cyclin E</td><td>Cyclin E</td><td>48</td></tr><tr><td>anti-Mad2</td><td>Mad2</td><td>25</td></tr><tr><td>anti-Pgk1</td><td>Pgk1</td><td>45</td></tr><tr><td>anti-Rad21/Scc1*</td><td>Rad21/Scc1 (subunit of cohesin)</td><td>68</td></tr></tbody></table></p><p><b>NOTE</b>: You can assume that all of these antibodies recognize the phosphorylated and unphosphorylated forms, if applicable, of each of the proteins. You will be able to see an appropriate shift in size for phosphorylated proteins on the western blot under the appropriate experimental conditions.</p><p>*The Rad21/Scc1 antibody recognizes a fragment of the Rad21 protein containing amino acids 423 – 501. Here is a schematic of the full length Rad21 protein (631 amino acids) with arrows indicating the amino acid positions of the two Separase cleavage sites:</p><img style =\'width:100%\' src=\'images/homepage/ps2_assignment.jpg\'/><p>III. Then, you visualize the cells’ DNA using immunofluorescence microscopy. Within StarCellBio, you will be able to perform one of the three following immunofluorescence analyses, as indicated within the microscopy setup page. For the siRNA #1-5 treatments, <b>only mitotic</b> images that are informative for the analysis of siRNA phenotypes have been provided. When analyzing microscopy images, make sure that you are comparing images generated by the same immunofluorescence microscopy analysis.<table class="scb_s_objective_table"><thead><td><b>Immunofluorescence Analysis</b></td><td><b>Description</b></td><td><b>Note</b></td></thead><tbody><tr><td>DAPI (blue), α-tubulin (green), γ-tubulin (red)</td><td>DAPI (4\',6-diamidino-2-phenylindole, a fluorescent stain that binds to DNA), α-tubulin antibody (a marker of microtubules), and γ-tubulin antibody (a marker of spindle poles)</td><td>The image available for control siRNA treated cells is a composite of representative images of cells in metaphase AND anaphase.</td></tr><tr><td>H2B (green), α-tubulin (red)</td><td>Histone 2B (H2B, a protein that co-localizes with DNA) antibody and an antibody to α-tubulin (a marker of microtubules)</td><td>The image available for control siRNA treated cells is of a cell in the same cell cycle stage as that of the siRNA experimental treatment images provided.</td></tr><tr><td>H2B (green)</td><td>Histone 2B (H2B, a protein that co-localizes with DNA) antibody</td><td>This immunofluorescence analysis is performed on metaphase cells after gently squashing the cells to enable attachments between sister chromatids to be visualized more readily.</td></tr></tbody></table></p><p>IV. Last, you visualize the expression and localization of the Mad2 protein. In StarCellBio, you may be able to perform the following immunofluorescence microscopy analysis, as indicated within the microscopy setup page. For the Mad2 microscopy analysis, only images of cells in <b>metaphase</b> have been provided.<table class="scb_s_objective_table"><thead><td><b>Immunofluorescence Analysis</b></td><td><b>Description</b></td></thead><tbody><tr><td>Mad2 (red), DAPI (blue)</td><td>An antibody to the Mad2 protein and DAPI (a fluorescent stain that binds to DNA)</td></tr></tbody></table></p><p>Conduct the appropriate experiments within StarCellBio to characterize the effect of each RNAi treatment, siRNA #1-5, on cell cycle progression. Using the results that you obtain from these experiments, answer the following questions.</p><p><b>NOTE:</b> Remember that to obtain full credit for this assignment, you are only required to submit answers to three questions of your choice out of the total five questions provided. Question 1 addresses the effect of siRNA #1, Question 2 addresses the effect of siRNA #2, etc. In addition, you are allowed to submit the two remaining questions for extra credit if they are completed in their entirety. For more information, see the “Getting Started” page of this assignment.</p></div>');
  return opt_sb ? '' : output.toString();
};


scb_assignment_specific_mit706s14ps2.page3 = function(opt_data, opt_sb) {
  var output = opt_sb || new soy.StringBuilder();
  output.append('<div class=\'scb_s_mit706s14ps2_assignment_overview\' xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"><h2>Questions 1-5</h2><p>Using your experimental results, answer the following questions within the MITx 7.06r website. Part A of this question will require you to consider only your flow cytometry experimental results, but for Parts B-E of this question, you should take into consideration all of your existing data when interpreting your experimental results for each individual part.</p><p>NOTE (for Question 2 only): Your advisor is concerned that one of the RNAi treatments, siRNA #2, actually contains two different siRNA constructs targeting two different gene products. Keep this potential concern in mind when analyzing the results following siRNA #2 treatment.</p><p><b class=\'ps2_bold\'>a)</b> Using only your flow cytometry data, indicate the cell cycle progression phenotype(s), if any, you observe in siRNA treated cells relative to control siRNA treated cells.</p><p><b class=\'ps2_bold\'>b)</b> Indicate the following for each protein in the table on the 7.06r website after siRNA treatment: (a) relative expression level in comparison to control siRNA treated cells, (b) the cleavage status, and (c) the phosphorylation status.</p><p><b class=\'ps2_bold\'>c)</b> Indicate the chromosome phenotype that best describes what you observe in siRNA treated cells.</p><p><b class=\'ps2_bold\'>d)</b> Indicate the Mad2 expression and localization phenotype that best describes what you observe in siRNA treated cells.</p><p><b class=\'ps2_bold\'>e)</b> Using all of your experimental results and answers from parts A-D, describe the phenotype you observe in siRNA treated cells. Your answer should include (i) the conclusion that you reached from analyzing ALL of your experimental results following siRNA treatment and (ii) the function of a potential gene product target for the siRNA. Be sure to explain the reasoning by which you arrived at your conclusion and reference your experimental results in your explanations.</p><br/><br/><br/></div>');
  return opt_sb ? '' : output.toString();
};


scb_assignment_specific_mit706s14ps2.experiment_setup = function(opt_data, opt_sb) {
  var output = opt_sb || new soy.StringBuilder();
  output.append('<ul class=\'scb_assignment_specific_tufts_experiment_setup\'><li>To setup your experiment, select <b>Add Samples</b> in the experimental set-up table below. <br></li><li>Select all of the treatment protocols for your experiment within the <b>Add Samples</b> pop up window, and then click <b>Add Samples</b>.<!--        Select all of the treatment protocols for your experiment and then click <b>Add Multiple Treatments</b>. --></li></ul>');
  return opt_sb ? '' : output.toString();
};
